app/selectors/emojis.js

```diff
@@ -38,24 +40,47 @@ function fillEmoji(indice) {
     };
 }
 
+// if an emoji
+// - has `skin_variations` then it uses the default skin (yellow)
+// - has `skins` it's first value is considered the skin version (it can contain more values)
```

Comment:
no, in the original data, there is only the default one (the yellow skinned one) and as a field it has skin_variations, from that we generate all the variations and add them the skins field which contains a list of skin tones for the emoji. If there is more than one person in it, like a couple, it will contain two items, but we use the first one to be the skin fo the emoji